.travis.yml

```diff
@@ -98,6 +104,12 @@ before_install:
       #now upgrade to avoid problems with outdated packages
       brew upgrade
 
+      brew install llvm
```

Comment:
llvm is quite large (1GB) and only used in one of the travis jobs. Do we want an if statement around this?

Reply:
ok. I think you can do
```bash
# check if we're using LLVM
if [[ $EXTRA_BUILD_FLAGS == *"llvm"* ]]; then
...
fi
```
Alternatively, you set `CC` and `CXX` as opposed to the CMake options, but why try that when the above should work...

Reply:
What I have works but truth be told, I dont know the correct `CC` and `CXX` flags for this. Would it be?

```
export CC=/usr/local/opt/llvm/bin/clang
export CXX=/usr/local/opt/llvm/bin/clang++
cmake {my_stuff}
```

Up to you which you prefer

Reply:
All the other travis jobs set `CC`/`CXX` in the job description. CMake will then use that.

However, your current set-up works, I don't see a reason to change it...